README.md

```diff
@@ -262,19 +244,6 @@ None
 >>> print(wv.get_vector("india", normalized=True))
 [-0.7871 -0.2993  0.3233 -0.2864  0.323 ]
 
-# Get the word vector for a word inidia.
->>> print(wv.get_vector('inidia', raise_exc=True))
```

Comment:
@RamanjaneyuluIdavalapati  @arrangeesh  why are the doc tests removed?

Reply:
@jaswanth098 Those doc tests are not valid. Because we don't have `raise_exc` parameter for the get_vector method.